iteration.go

```diff
@@ -90,6 +90,48 @@ func (c *IterationController) Show(ctx *app.ShowIterationContext) error {
 	})
 }
 
+// Update runs the update action.
+func (c *IterationController) Update(ctx *app.UpdateIterationContext) error {
+	_, err := login.ContextIdentity(ctx)
```

Comment:
@baijum I tried removing this but `TestFailUpdateIterationUnauthorized` fails with -
```
iteration_testing.go:1085: invalid response status code: got 200, expected 401
```
because `UpdateIterationUnauthorized` generated by goa and defined in `app/test/iteration_testing.go` is expecting controller to return unauthorized response. Hence I need to keep that part as it is.